### PR TITLE
Include HEAD when browsing ancestors

### PR DESCRIPTION
**Pull request by charlietran**

### CODE REVIEW

File: GitUp/Application/en.lproj/MainMenu.xib [hunk 1]
```diff
@@ -594,7 +594,7 @@ You must close and reopen any opened repositories in GitUp after changing this s
                                 </connections>
                             </menuItem>
                             <menuItem isSeparatorItem="YES" id="qQz-Pc-UPf"/>
-                            <menuItem title="Browse HEAD Ancestors…" keyEquivalent="d" id="6tA-ab-KI4">
+                            <menuItem title="Browse HEAD Commits…" keyEquivalent="d" id="6tA-ab-KI4">
```
Comment (swisspol) [inline GitUp/Application/en.lproj/MainMenu.xib]: I'm not sure "Browse HEAD Commits" make sense. There is only one HEAD commit by definition. The original label is still better after thinking about it.

Comment (charlietran) [inline GitUp/Application/en.lproj/MainMenu.xib]: Ah you're right, will change it back!

**Comment by swisspol** [comment]
So you tested this extensively? What happens if the repo is empty and has no HEAD?

**Comment by charlietran** [comment]
In an empty repo, the `Browse HEAD Ancestors...` command is unavailable:

[](https://www.dropbox.com/s/kkfk5ft3a2vh08u/Screenshot%202015-08-27%2011.48.33.png?dl=1)

**Comment by charlietran** [comment]
Also I did test this across several repos and different branches. Any edge cases with missing HEADs seem to be accounted for.

**Comment by swisspol** [comment]
All good then :+1:

**Comment by swisspol** [comment]
@charlietran Can you take a minute to agree to the new GitUp CLA: http://forums.gitup.co/t/gitup-contributor-license-agreement/323? Thanks!
